### ISSUE TITLE
Centered Images

### ISSUE THREAD

**Issue by GoogleCodeExporter** [birdal46/grafx2#507]
```
Would it be possible to have a mode where the image is centered if it's smaller 
than the current work area.

F.ex. I have noticed that my current monitor have quite variable brightnesses 
in different colorregisters depending on positions (i.e view angles). This 
makes colorprecise work and evaluation very hard and unreliable - as a smaller 
image is displayed at top-right (more error) rather than the center of the 
screen.

And it might be more convinient for image-editing in general.
```

Original issue reported on code.google.com by `annas...@hotmail.com` on 18 Sep 2012 at 9:36
- Merged into: #96

**Comment by GoogleCodeExporter** [comment]
```
Yes it would be better. It's been mentioned a few times, the only reason 
Pulkomandy and I haven't started looking into it is that it affects a very 
large part of the code. Some impacts, off the top of my head :
- clicking "outside the image bounds" is only checked on the right and bottom 
border
- The mouse position in image coordinates (Mouse_X&Y) is unsigned and going to 
overflow
- many graphic functions (drawing in image space, drawing part of the image to 
screen) use unsigned integers and are going to overflow.
Basically, every coordinate defined as a "word" should be checked. If it's a 
coordinate in image space, it'll likely need to be changed.

```

Original comment by `yrizoud` on 18 Sep 2012 at 12:13
- Changed state: **Duplicate**
